spec/features/hq/user_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+RSpec.feature 'User CRUD spec:', :type => :feature do
+  
+  background do
+    i_sign_in_as_admin
+    an_user_exist 98, 'First User', 'first@example.com'
+    an_user_exist 99, 'Second User', 'second@example.com'
+  end
+  
+  scenario 'There should be a link to the Users page on the navbar' do
+    visit hq_root_path
+    expect(page).to have_link( 'Users', href: hq_users_path )
+  end
+  
+  scenario 'There should be a list of users on the users index page' do
+    visit hq_users_path
+    and_i_should_see 'First User'
+    and_i_should_see 'Second User'
+  end
+  
+  scenario 'The index page should show the number of active sponsors assigned to a user' do
+    a_sponsorship_exist( 98 )
+    visit hq_users_path
+    and_i_should_see '1'
```

Comment:
I think this `1` is dangerous. It could check for different things too. Better replace it with something more specific.
Also are the parenthesis needed on L23?